### DISCUSSION TITLE
If we could design web UI  by dragging the element ?

### DISCUSSION THREAD

**Discussion by v479038280** [zauberzeug/nicegui#1828] [Ideas / Feature Requests]
If we could design Web UI like Qtdesinger?

**Comment by stdusr** [comment]
There were similar requests, e.g. https://github.com/zauberzeug/nicegui/discussions/688
If you need drag and drop UI builder, you can have a look at:
https://www.abstra.io/editor
https://docs.atrilabs.com
https://anvil.works
https://github.com/streamsync-cloud/streamsync

**Comment by falkoschindler** [comment]
Sorry, I'll close this discussion as a duplicate of https://github.com/zauberzeug/nicegui/discussions/688.
Thanks for the reference, @stdusr!

**Reply by v479038280** [reply]
https://github.com/streamsync-cloud/streamsync
This tool is so cool.hope nicegui could integrate  such function as an alternaive
